components/badge/badge.mdx

```diff
@@ -65,6 +65,10 @@ import { DtBadge } from '@dialpad/dialtone-vue';
 ```jsx
 <dt-badge
   text="badge"
+  decoration="default | black-400 | black-500 | black-900 | red-200 | red-300 | red-400
```

Comment:
You can just remove "default" here since users will just not include the prop if they don't want it.

Reply:
If "default" is not necessary then I would recommend to set the default value of the propr decoration to `null` and remove `default` also from BADGE_DECORATION_MODIFIERS